demo/.strapi-app/scripts/seed.js

```diff
@@ -266,6 +266,11 @@ async function createApiTokens(app) {
     { name: 'Read Only Token', type: 'read-only' },
   ];
 
+  const envPaths = [
```

Comment:
So this will need to be updated every time we add/rename/remove a demo app, correct?

Reply:
yeah, should only need it one or two more times for nextjs and maybe simple react?

Reply:
We'll also need a browser one for the raw html/js

Reply:
What we can do then is add a small section about demos in the readme? Or even a readme inside the demo folder?

Basically something where we explain the goal of those demo, how to work with them, add or remove one, etc...